requirements.txt

```diff
@@ -0,0 +1,12 @@
+argparse==1.4.0
```

Comment:
I don't think these should be == requirements but instead >= requirements.   ???